website/docs/r/multiClusterApp.html.markdown

```diff
@@ -3,16 +3,16 @@ layout: "rancher2"
 page_title: "Rancher2: rancher2_multi_cluster_app"
 sidebar_current: "docs-rancher2-resource-multi_cluster_app"
 description: |-
-  Provides a Rancher v2 multi_cluster_app resource. This can be used to deploy multi cluster apps on rancher v2.
+  Provides a Rancher v2 multi_cluster_app resource. This can be used to deploy multi cluster apps on Rancher v2.
 ---
 
 # rancher2\_multi_cluster_app
 
-Provides a Rancher v2 multi_cluster_app resource. This can be used to deploy multi_cluster_app on rancher v2.
+Provides a Rancher v2 multi_cluster_app resource. This can be used to deploy multi_cluster_app on Rancher v2.
 
-This resource can also modify rancher v2 multi cluster apps in 3 ways:
-- `Add/Remove targets`: If `targets` arguments is modified, the multi cluster app targets will be updated. 
-- `Rollback`: If `revision_id` argument is provided or modified the multi cluster app will be rollbacked accordingly. It will generate a non empty tf plan that will require manual tf file edition. Use carefully.
+This resource can also modify Rancher v2 multi cluster apps in 3 ways:
+- `Add/Remove targets`: If `targets` arguments is modified, the multi cluster app targets will be updated.
+- `Rollback`: If `revision_id` argument is provided or modified the multi cluster app will be rolled back accordingly. It will generate a non-empty tf plan that will require manual .tf file intervention. Use carefully.
```

Comment:
Using same format used at `website/docs/r/app.html.markdown `
```
- `Rollback`: If `revision_id` argument is provided or modified the multi cluster app will be rolled back accordingly. It will generate a non-empty tf plan that will require manual .tf file intervention. Use carefully.
- `Rollback`: If `revision_id` argument is provided or modified the multi cluster app will be rolled back accordingly. It will generate a non-empty terraform plan that will require manual .tf file intervention. Use carefully.
```

Reply:
fixed